common.gypi

```diff
@@ -36,7 +36,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.10',
+    'v8_embedder_string': '-node.11',
```

Comment:
just to know, will this have to be 12? if this one lands https://github.com/nodejs/node/pull/47209 to prevent conflicts?

Reply:
Depending on which one lands first we can just edit the number during landing.